lib/srv/server/ssm_install.go

```diff
@@ -65,11 +71,27 @@ type SSMRunRequest struct {
 	AccountID string
 }
 
+// CheckAndSetDefaults ensures the emitter is present and creates a default logger if one is not provided.
+func (c *SSMInstallerConfig) CheckAndSetDefaults() error {
```

Comment:
Could we make this private? I assume the it's NewSSMInstaller's responsibility to always call it.